src/librustc/metadata/tydecode.rs

```diff
@@ -924,41 +915,35 @@ fn parse_builtin_bounds_<F>(st: &mut PState, _conv: &mut F) -> ty::BuiltinBounds
     }
 }
 
-fn parse_bounds<'a, 'tcx, F>(st: &mut PState<'a, 'tcx>, mut conv: F)
-                             -> ty::ParamBounds<'tcx> where
-    F: FnMut(DefIdSource, ast::DefId) -> ast::DefId,
-{
-    parse_bounds_(st, &mut conv)
-}
-
-fn parse_bounds_<'a, 'tcx, F>(st: &mut PState<'a, 'tcx>, conv: &mut F)
-                              -> ty::ParamBounds<'tcx> where
+fn parse_bounds<'a, 'tcx, F>(st: &mut PState<'a, 'tcx>, conv: &mut F)
```

Comment:
For conventions sake I should probably change this back to `parse_bounds_`, didn't realize the convention until I was looking over the change set.